Doc/library/statistics.rst

```diff
@@ -559,6 +570,93 @@ However, for reading convenience, most of the examples show sorted sequences.
 
    .. versionadded:: 3.8
 
+.. function:: covariance(x, y, /)
+
+   Calculates covariance of two variables *x* and *y*. Covariance is
+   a measure of the joint variability of two variables.
+
+   Raises :exc:`StatisticsError` if both variables have same number of data
```

Comment:
This seems to be reversed: An exception would  be raised if the number of data points were different, right?

(The same goes fro the docs of the following function as well.)